pkg/commands/rules.go

```diff
@@ -271,6 +274,13 @@ func (r *RuleCommand) setupFiles() error {
 		}
 	}
 
+	// Set up rule groups excluded from label aggregation.
+	for _, name := range strings.Split(r.AggregationLabelExcludedRuleGroups, ",") {
+		if name = strings.TrimSpace(name); name != "" {
+			r.aggregationLabelExcludedRuleGroupsList = append(r.aggregationLabelExcludedRuleGroupsList, name)
```

Comment:
nit: you could use a set here instead of `[]string` for faster lookups in `applyTo`

Reply:
You're right. Done!